ksp_plugin_adapter/localization/zh-cn.cfg

```diff
@@ -73,16 +77,20 @@ Localization {
     #Principia_ReferenceFrameSelector_Description_BodyCentredNonRotating = <<1>>的质心已固定，\n水平面为它的赤道。\n这个参考系不旋转。
     #Principia_ReferenceFrameSelector_Description_BodyCentredParentDirection = <<1>>的质心已固定，\n水平面为绕着<<3>>的轨道。\n这个参考系跟随轨道旋转，面向着<<3>>的方位不变。
     #Principia_ReferenceFrameSelector_Description_BodySurface = <<1>>已固定，\n水平面为它的赤道。\n这个参考系与<<1>>共同旋转。
+    #Principia_ReferenceFrameSelector_Description_RotatingPulsating = <<3>>的质心与<<1>>的质心已固定。\n水平面为它们轨道的平面。\n这个参考系与轨道同步旋转并且脉动，以使<<3>>-<<1>>距离恒定。
+    #Principia_ReferenceFrameSelector_Description_RotatingPulsatingOneMoon = <<3>>的质心与<<1>>-<<5>>系统质心已固定。\n水平面为它们轨道的平面。\n这个参考系与轨道同步旋转并且脉动，以使<<3>>-<<1>>距离恒定。
+    #Principia_ReferenceFrameSelector_Description_RotatingPulsatingBigSystem = <<3>>的质心与<<1>>系统质心已固定。\n水平面为它们轨道的平面。\n这个参考系与轨道同步旋转并且脉动，以使<<3>>-<<1>>距离恒定。
```

Comment:
@al2me6 These need to be updated to say something about the line between <<3>> and <<1>>.

Reply:
```suggestion
    #Principia_ReferenceFrameSelector_Description_RotatingPulsating = <<3>>的质心与<<1>>的质心已固定。\n水平面为它们轨道的平面。\n这个参考系与轨道旋转，以使<<3>>-<<1>>连线固定，并且脉动，以使<<3>>-<<1>>距离恒定。
    #Principia_ReferenceFrameSelector_Description_RotatingPulsatingOneMoon = <<3>>的质心与<<1>>-<<5>>系统质心已固定。\n水平面为它们轨道的平面。\n这个参考系与轨道旋转，以使<<3>>-<<1>>连线固定，并且脉动，以使<<3>>-<<1>>距离恒定。
    #Principia_ReferenceFrameSelector_Description_RotatingPulsatingBigSystem = <<3>>的质心与<<1>>系统质心已固定。\n水平面为它们轨道的平面。\n这个参考系与轨道旋转，以使<<3>>-<<1>>连线固定，并且脉动，以使<<3>>-<<1>>距离恒定。
```